--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -101,7 +101,10 @@
 # You should not enable it unless you know what you are doing.
 option(KIMAGEFORMATS_WITH_KNOWN_CRASHES_JXR "Enable plugin for JPEG XR format" OFF)
 if(KIMAGEFORMATS_WITH_KNOWN_CRASHES_JXR)
-    find_package(LibJXR)
+    find_package(JXR)
+    set(LibJXR_FOUND ${JXR_FOUND})
+    set(LIBJXR_INCLUDE_DIRS ${JXR_INCLUDE_DIRS})
+    set(LIBJXR_LIBRARIES ${JXR_LIBRARIES})
 endif()
 add_feature_info(LibJXR LibJXR_FOUND "required for the QImage plugin for JPEG XR images")
 
