--- a/portfile.cmake
+++ b/portfile.cmake
@@ -2,9 +2,9 @@
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO 4creators/jxrlib
-    REF f7521879862b9085318e814c6157490dd9dbbdb4
-    SHA512 f5617cbe73b6b905cc6bba181e6a3efedd59584f7a8c90e0f34db580cfdad4239a2ab753df4e221f26a5c0db51475b021052e3b9e3ab3673573573b1d57f3fdb
+    REPO mircomir/jxrlib
+    REF 14b78eafa4848d63d8fc32792cc398f3b8c4ae84
+    SHA512 b2eea27ef0d9fb4ff95159a71df6cf2927482e2c0e64545e1a820c17cf95f752253b845fe1bf4e1a62ef94e4499a73ea93c3a2c77b9fdeff9c0c809b5fb8d339
     HEAD_REF master
     PATCHES
         guiddef.patch
