--- a/portfile.cmake
+++ b/portfile.cmake
@@ -2,9 +2,9 @@
 
 vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
-    REPO 4creators/jxrlib
-    REF f7521879862b9085318e814c6157490dd9dbbdb4
-    SHA512 f5617cbe73b6b905cc6bba181e6a3efedd59584f7a8c90e0f34db580cfdad4239a2ab753df4e221f26a5c0db51475b021052e3b9e3ab3673573573b1d57f3fdb
+    REPO mircomir/jxrlib
+    REF 72785650541746b85d612cfc7e1ef4da3ca8bab8
+    SHA512 49297466ccdaecc3bca21406c36c38e739835e993003a5c0725fdae55ee7eea15e8fd521c735d68a995111aebed9a2c0d805bf309d3c1583f5089ffb69791f42
     HEAD_REF master
     PATCHES
         guiddef.patch
