diff --git a/common/include/guiddef.h b/common/include/jxrguiddef.h
similarity index 100%
rename from common/include/guiddef.h
rename to common/include/jxrguiddef.h
diff --git a/jxrgluelib/JXRGlue.h b/jxrgluelib/JXRGlue.h
index 7ed587e..938e122 100644
--- a/jxrgluelib/JXRGlue.h
+++ b/jxrgluelib/JXRGlue.h
@@ -33,7 +33,11 @@ extern "C" {
 
 #include <JXRMeta.h>
 #include <JXRVersion.h>
+#ifdef _WIN32
 #include <guiddef.h>
+#else
+#include <jxrguiddef.h>
+#endif
 
 //================================================================
 #define WMP_SDK_VERSION 0x0101
