diff --git a/CMakeLists.txt b/CMakeLists.txt
index a28e6c3..e3be6f3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -616,6 +616,8 @@ if(BUILD_SHARED_LIBS)
     if(AVIF_LIB_USE_CXX)
         set_target_properties(avif PROPERTIES LINKER_LANGUAGE "CXX")
     endif()
+else()
+    target_include_directories(avif_static PUBLIC $<BUILD_INTERFACE:${libavif_SOURCE_DIR}/include> $<INSTALL_INTERFACE:include>)
 endif()
 
 # Give access to functions defined in internal.h when BUILD_SHARED_LIBS is ON, to tests for example.
